lib/Sema/TypeCheckAccess.h

```diff
@@ -17,6 +17,8 @@
 #ifndef TYPECHECKACCESS_H
 #define TYPECHECKACCESS_H
 
+#include "swift/AST/Decl.h"
```

Comment:
It is customary to forward-declare types in headers instead, eg put 'class Decl;' inside the 'namespace swift' block

Reply:
It's now updated.